crates/polars-utils/src/io.rs

```diff
@@ -7,13 +8,8 @@ pub fn open_file<P>(path: P) -> PolarsResult<File>
 where
     P: AsRef<Path>,
 {
-    std::fs::File::open(&path).map_err(|e| {
+    std::fs::File::open(&path).map_err(|err| {
         let path = path.as_ref().to_string_lossy();
-        if path.len() > 88 {
```

Comment:
Can we keep this logic? This was there to ensure we spit out the sensible last part of a very long path.

Reply:
All right I can reinstate that logic :+1: I couldn't think of an instance where viewing the full path would be detrimental. I checked the Python native error messages and those print the whole thing.

Reply:
This was the issue that added this. If we want to remove this, I don't think it should be done as a drive-by.

https://github.com/pola-rs/polars/issues/6853

Reply:
I added back the path truncation 👍